explainaboard/tests/test_extractive_qa.py

```diff
@@ -33,6 +34,10 @@ def test_generate_system_analysis(self):
         # analysis.write_to_directory("./")
         self.assertIsNotNone(sys_info.results.fine_grained)
         self.assertGreater(len(sys_info.results.overall), 0)
+        # should be 0.6974789915966386
+        self.assertGreater(sys_info.results.overall["ExactMatchQA"].value, 0.69)
```

Comment:
Maybe `assertAlmostEqual` is more appropriate?

Reply:
>"As an alternative suggestion, maybe w..."

I actually like this idea and will try to go towards this end.


>"Maybe assertAlmostEqual is more appropriate?"

Interesting! it seems that python has provided everything...

Reply:
This still needs to be fixed I guess.